proto/cosmwasm/wasm/v1/types.proto

```diff
@@ -20,10 +20,6 @@ enum AccessType {
   // AccessTypeNobody forbidden
   ACCESS_TYPE_NOBODY = 1
       [ (gogoproto.enumvalue_customname) = "AccessTypeNobody" ];
-  // AccessTypeOnlyAddress restricted to a single address
-  // Deprecated: use AccessTypeAnyOfAddresses instead
-  ACCESS_TYPE_ONLY_ADDRESS = 2
```

Comment:
Can you add a `reserved 2; // was AccessTypeOnlyAddress` to the file, for documentation?